airdrop/job/reward_processors/nunet_reward_processor.py

```diff
@@ -99,7 +116,9 @@ def process_rewards(self):
                 self._users_to_reward.append(u)
                 sum_of_log_values += getattr(u,"_log10_score")
         
-        logger.info(f"Normalized Score {sum_of_log_values} for {len(self._users_to_reward)}")
+        score_message = f"For window {self._window_id}, Normalized Score is {sum_of_log_values} for {len(self._users_to_reward)}. Skipped users {len(self._users_to_reward) - len(self._users_to_reward)}"
```

Comment:
is the scrip re re runable  ? say it fails midway , can we pick it from where we left ? or will it have to be re run entirely ?

Reply:
Can we also please take a back up of the rewards and do an additional sanity check to make sure that any deltas have lesser rewards ONLY

Reply:
Script has to be re-run always. Key updates happen in a single transaction is its an all or nothing operation.
What do you mean back up of rewards?

Reply:
take a back up of the user_rewards table before ruuning the scrip